app/src/main/res/layout/note_element.xml

```diff
@@ -27,7 +28,7 @@
         app:layout_constraintRight_toRightOf="@+id/title_note_element"
         app:layout_constraintTop_toBottomOf="@+id/title_note_element"
         tools:text="This is a great text to check if the textview will break asdsadsadsad"
-        android:textIsSelectable="true"/>
+        android:textIsSelectable="false"/>
```

Comment:
I've added this property to fix lint issue, but later I thought it would be good to let user select text and reuse it on other note. Did you see issues giving user possibility to select text?

Reply:
Yes ... if we set this property to "true" value, the TextView will intercept the touch and in this case, the touch in one note will note works properly.

Reply:
hmmm got it.